backend/app/graphql/auth.py

```diff
@@ -0,0 +1,131 @@
+import graphene
+
+from .types import Mutation, MutationList
+from ..graphql.services import services
+from ..resources.create_user_dto import CreateUserDTO
+
+
+class User(graphene.ObjectType):
+    access_token = graphene.String()
+    id = graphene.ID()
+    first_name = graphene.String()
+    last_name = graphene.String()
+    email = graphene.String()
+    role = graphene.String()
+
+
+class Login(Mutation):
+    """
+    Returns access token in response body and sets refreshToken as an httpOnly cookie
+    """
+    class Arguments:
+        email = graphene.String()
+        password = graphene.String()
+        id_token = graphene.String()
+
+    user = graphene.Field(User)
+
+    def mutate(self, info, email, password, id_token):
+        auth_dto = None
+        if id_token:
+            auth_dto = services["auth_service"].verify_token(id_token)
```

Comment:
is the `verify_token` method defined? i can't seem to find it in `auth_service.py`